src/components/EditForm.jsx

```diff
@@ -7,13 +7,23 @@ import { GrUpdate } from "react-icons/gr";
 
 import "./common.css"
 
-const EditForm = ({ id, title, description }) => {
+const EditForm = ({ note }) => {
+    const { id, title, description } = note
     const { dispatch } = useNote();
     const [isEditing, setIsEditing] = useState(false);
     const [editFrom, setEditForm] = useState({
         updatedTitle: title,
         updatedDescription: description,
     });
+
+    useEffect(() => {
+        if (note) {
+            setEditForm(prev => ({
+                ...prev
+            }))
```

Comment:
You're just spreading the previous state and setting it without any change.
What exactly is the intention here?